queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -57,7 +58,7 @@ def _get_pdfs(
         if filing_pdf_encoded:
             pdfs.append(
                 {
-                    'fileName': 'Continuation Application.pdf',
+                    'fileName': 'Continuation Application - Pending.pdf',
```

Comment:
This should be ok, right @vysakh-menon-aot ?

This is the case where the continuation application is "PAID", which matches the email I've been working on.